### DISCUSSION TITLE
TuringCourses/major_module/distributed_computing/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#94] [General]
# TuringCourses/major_module/distributed_computing/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_module/distributed_computing/

**Comment by NeilAlfred** [comment]
别看了，到 2023 年这门课还是门僵尸课，培养方案上有，但没人开。

**Comment by Dreaming-Q** [comment]
必看了，2025年还是僵尸课
